Packs/Core/Playbooks/playbook-Uncommon_remote_scheduled_task_created.yml

```diff
@@ -0,0 +1,1352 @@
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "66":
+    id: "66"
+    taskid: f5c5e77b-66e5-465a-8773-c1d20a200bfa
+    type: condition
+    task:
+      id: f5c5e77b-66e5-465a-8773-c1d20a200bfa
+      version: -1
+      name: Found any malicious or suspicious cmd parameters?
+      description: Determines the appropriate verdict based on the results of the command-line analysis.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      Malicious Cmd parameters:
+      - "3"
+      Suspicious Cmd parameters:
+      - "41"
+    separatecontext: false
+    conditions:
+    - label: Malicious Cmd parameters
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.AMSI
+            iscontext: true
+          right:
+            value: {}
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.maliciousTools
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.networkActivity
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.SuspiciousLolbinExecution
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.SuspiciousCmdPathAndArguments
+            iscontext: true
+    - label: Suspicious Cmd parameters
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.base64
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: CommandlineVerdict.suspiciousParameters
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1070,
+          "y": 1800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "67":
+    id: "67"
+    taskid: e7cb4db3-f70e-4474-8ae5-1ad159731138
+    type: regular
+    task:
+      id: e7cb4db3-f70e-4474-8ae5-1ad159731138
+      version: -1
+      name: War room notification - Scheduled Task Disabled
+      description: Prints text to war room (Markdown supported)
+      scriptName: Print
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "23"
+    scriptarguments:
+      value:
+        simple: "Dear Analyst,\n\nPlease note that during the remediation process, the playbook executed a shell command to disable the following scheduled task: \n${Core.OriginalAlert.event.scheduled_task_path}\n\n"
```

Comment:
"Dear Analyst,\n\nDuring the remediation process the playbook executed a shell command to disable the following scheduled task: \n${Core.OriginalAlert.event.scheduled_task_path}\n\n"